Mlem/Views/Shared/Instance/InstanceStatsView.swift

```diff
@@ -0,0 +1,270 @@
+                        "Federation Signed Fetch",
+                        systemImage: Icons.developerMode,
+                        value: signedFetch
+                    )
+                }
+            }
+            .frame(maxWidth: .infinity)
+            .background(Color(uiColor: .secondarySystemGroupedBackground))
+            .cornerRadius(AppConstants.largeItemCornerRadius)
+            
+            VStack(alignment: .leading, spacing: 0) {
+                settingRow(
+                    "Registration",
+                    systemImage: Icons.person,
+                    value: instance.registrationMode?.label ?? "Closed",
+                    color: instance.registrationMode?.color ?? .red
+                )
+                if instance.registrationMode != .closed {
+                    Divider()
+                    settingRow(
+                        "Email Verification",
+                        systemImage: Icons.email,
+                        value: instance.requiresEmailVerification ?? false
+                    )
+                    Divider()
+                    settingRow(
+                        "Captcha",
+                        systemImage: Icons.photo,
+                        value: captchaLabel,
+                        color: instance.captchaDifficulty == nil ? .red : .green
+                    )
+                }
+            }
+            .frame(maxWidth: .infinity)
+            .background(Color(uiColor: .secondarySystemGroupedBackground))
+            .cornerRadius(AppConstants.largeItemCornerRadius)
+            
+            VStack(alignment: .leading, spacing: 0) {
+                settingRow(
+                    "NSFW Content",
+                    systemImage: Icons.blurNsfw,
+                    value: instance.allowsNSFW ?? false
+                )
+                Divider()
+                settingRow(
+                    "Downvotes",
+                    systemImage: Icons.downvote,
+                    value: instance.allowsDownvotes ?? false
+                )
+                Divider()
+                settingRow(
+                    "Community Creation",
+                    systemImage: "house",
+                    value: instance.allowsCommunityCreation ?? false
+                )
+                Divider()
+                settingRow(
+                    "Slur Filter",
+                    systemImage: Icons.filterFill,
+                    value: instance.slurFilterRegex != nil
+                )
+                if developerMode, let regex = instance.slurFilterString {
+                    Divider()
+                    Text(showingSlurRegex ? regex : "Tap to show slur filter regex")
+                        .textSelection(.enabled)
+                        .font(.footnote)
+                        .foregroundStyle(.secondary)
+                        .padding(12)
+                        .onTapGesture {
+                            showingSlurRegex.toggle()
```

Comment:
This might want a `.withAnimation` just to smooth it out a little

Reply:
Sure